docker/Dockerfile

```diff
@@ -1,27 +1,34 @@
-FROM rust:latest
+# Full build with debuginfo for graph-node
+#
+# The expectation if that the docker build uses the parent directory as PWD
+# by running something like the following
+#   docker build --target STAGE -f docker/Dockerfile .
 
-# Replace this with the graph-node branch you want to build the image from;
-# Note: Docker Hub substitutes this automatically using our hooks/post_checkout script.
-ENV SOURCE_BRANCH "master"
```

Comment:
I'm not sure why I wrote "our" `hooks/post_checkout` script, but Docker Hub I think needs the `SOURCE_BRANCH` environment variable to pick the branch / commit to build.

Reply:
Now that we are building in Cloud Build, this is moot anyway

Reply:
True